src/components/Shared/Property.js

```diff
@@ -0,0 +1,177 @@
+<<<<<<< HEAD:src/components/Shared/Property.js
```

Comment:
this is the reason it again fails, bro resolve all conficts before pushing, just try to run the code it will fails,in these cases,

Reply:
checked, and ran entire code. it works,
is the server not working? because the properties are not rendering quickly. Tell me if there is any other issue. sorry for this delay

Reply:
Cool I will fix it